ext/strings_test.go

```diff
@@ -879,15 +878,17 @@ func TestStringFormat(t *testing.T) {
 			),
 		}
 		if locale != "" {
-			opts = append([]cel.EnvOption{cel.Locale(locale)}, opts...)
+			opts = append([]cel.EnvOption{Strings(StringsLocale(locale))}, opts...)
```

Comment:
I think you can always set `StringsLocale()` since an empty value for `locale` will just leave the string formatting unaltered.

Reply:
Right, I've removed that logical branch now.